tools/utils.py

```diff
@@ -10,15 +9,3 @@ def get_credentials_from_netrc(*, hostname, netrc_file=None):
         return None, None
 
     return credentials[0], credentials[2]
-
-
-def normalize_jira_title(s):
```

Comment:
I expect all of this to be irrelevant, now that all new issues are normalized (and we only look at last month's issues)